src/main/java/org/apache/datasketches/req/ReqCompactor.java

```diff
@@ -49,7 +49,6 @@ class ReqCompactor {
   //objects
   private FloatBuffer buf;
   private final ReqDebug reqDebug = null;
-  private Random rand;
```

Comment:
I'm surprised SpotBugs complained about the use of random here

Reply:
Actually, it was complaining that an instance of Random was created and then used only once.  This is not only slow, but it does not produce good random behavior.  

So I took a hard look at how Random was used in REQ and refactored the use of Random in REQ completely.  Now, Random is instantiated once in ReqSketch and then a reference to it is passed to the ReqCompactor.compact(...) method which then uses the same random instance for every compaction.  This also opens the door for us to easily add a "makeDeterministic" flag to the public constructor if we decide we want to do that (which I am increasing thinking that this would be ok to do).